common/on-chip-calib.h

```diff
@@ -10,118 +10,120 @@
-        int speed = 3;
-        bool tare = false;
-        bool intrinsic_scan = true;
-        bool apply_preset = true;
-
-        void calibrate();
-
-    private:
-
-        std::vector<uint8_t> safe_send_command(const std::vector<uint8_t>& cmd, const std::string& name);
-
-        rs2::depth_frame fetch_depth_frame(invoker invoke);
-
-        std::pair<float, float> get_depth_metrics(invoker invoke);
-
-        void process_flow(std::function<void()> cleanup, invoker invoke) override;
-
-        float _health = -1.0f;
-        device _dev;
-
-        bool _was_streaming = false;
-        bool _synchronized = false;
-        bool _post_processing = false;
-        std::shared_ptr<subdevice_ui_selection> _ui { nullptr };
-        bool _in_3d_view = false;
-
-        viewer_model& _viewer;
-        std::shared_ptr<subdevice_model> _sub;
-
-        std::vector<uint8_t> _old_calib, _new_calib;
-        std::vector<std::pair<float, float>> _metrics;
-        device_model& _model;
-
-        bool _restored = true;
-
-        void stop_viewer(invoker invoke);
-        void start_viewer(int w, int h, int fps, invoker invoke);
-    };
-
-    // Auto-calib notification model is managing the UI state-machine
-    // controling auto-calibration
-    struct autocalib_notification_model : public process_notification_model
-    {
-        enum auto_calib_ui_state
-        {
-            RS2_CALIB_STATE_INITIAL_PROMPT,  // First screen, would you like to run Health-Check?
-            RS2_CALIB_STATE_FAILED,          // Failed, show _error_message
-            RS2_CALIB_STATE_COMPLETE,        // After write, quick blue notification
-            RS2_CALIB_STATE_CALIB_IN_PROCESS,// Calibration in process... Shows progressbar
-            RS2_CALIB_STATE_CALIB_COMPLETE,  // Calibration complete, show before/after toggle and metrics
-            RS2_CALIB_STATE_TARE_INPUT,      // Collect input parameters for Tare calib
-            RS2_CALIB_STATE_TARE_INPUT_ADVANCED,      // Collect input parameters for Tare calib
-            RS2_CALIB_STATE_SELF_INPUT,      // Collect input parameters for Self calib
-        };
-
-        autocalib_notification_model(std::string name,
-            std::shared_ptr<on_chip_calib_manager> manager, bool expaned);
-
-        on_chip_calib_manager& get_manager() { 
-            return *std::dynamic_pointer_cast<on_chip_calib_manager>(update_manager); 
-        }
-
-        void set_color_scheme(float t) const override;
-        void draw_content(ux_window& win, int x, int y, float t, std::string& error_message) override;
-        void draw_dismiss(ux_window& win, int x, int y) override;
-        void draw_expanded(ux_window& win, std::string& error_message) override;
-        void draw_intrinsic_extrinsic(int x, int y);
-        int calc_height() override;
-        void dismiss(bool snooze) override;
-
-        bool use_new_calib = true;
-        std::string _error_message = "";
-    };
+	class viewer_model;
```

Comment:
Please check why git reports the whole file as changed
If there are no intentional changes please undo changes to this file